File: GotDotNet.com/UrlBuddy/VB.NET/UrlInfo.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Public Class UrlInfo

    Public Url As String
    Public Title As String
    
    Public Sub New()
    End Sub

    Public Sub New(url As String)
        MyClass.New(url, url)
    End Sub

    Public Sub New(ByVal aUrl As String, ByVal aTitle As String)
        Url = aUrl
        Title = aTitle
    End Sub

End Class

File: GotDotNet.com/UrlBuddy/C#/UrlInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UrlBuddy
{
	public class UrlInfo
	{
		public string Url;
		public string Title;

		public UrlInfo()
		{}

		public UrlInfo(string url) : this(url, url)
		{}

		public UrlInfo(string url, string title)
		{
			Url = url;
			Title = title;
		}
	}
}
